source/infrastructure/lib/aws-instance-scheduler-stack.ts

```diff
@@ -686,6 +700,10 @@ export class AwsInstanceSchedulerStack extends cdk.Stack {
         ]
       }
     }
+    NagSuppressions.addResourceSuppressions(lambdaToDynamoDb, [{
+      id: "AwsSolutions-IAM5",
+      reason: "This Lambda function has permission provided to write to CloudWatch logs using the iam roles."
+    }], true)
```

Comment:
Not a big fan of applying to every child resource. This gets added to some strange things like the state table. You should probably go into the construct and add the required suppressions on the specific resources. When adding to a policy, review what the policy is for and explain. Someone is going to look at the suppression reason in a csv file, so it should generally explain why the permission is needed.